### ISSUE TITLE
Changing playback speed doesn't work on own screencasts

### ISSUE THREAD

**Issue by bobziroll** [scrimba/community#468]
If I'm logged in to Scrimba, I'm unable to speed up the screencast. The indicator says it should be playing faster, but it doesn't actually do so. If I log out or use incognito mode it works as expected, and it works if I'm watching a screencast recorded by someone else while logged in.

**Comment by rokublac** [comment]
I ran into a similar issue but I had to pause and play the cast again to activate the effect. Although the speed change worked, it did cause a pitch shift. 

So on my test `Chromium 79.0.3945.79`, when viewing your own cast:
- Speed function not activating on its value change but rather from pause and play.
- Change in speed causes pitch shift.

Are you also running into this issue?

**Comment by bobziroll** [comment]
Yes, that's the same thing I'm seeing now too. Chrome 79.0.3945.88 for me right now.

**Comment by rokublac** [comment]
> Yes, that's the same thing I'm seeing now too, both in Chrome and Firefox.
> 
> Chrome 79.0.3945.88
> Firefox 71.0

Okay, I'll edit this issue to reflect the current problem. Thanks

**Comment by perborgen** [comment]
When you're watching your own tutorials, you'll always hear a pitch shift when speeding up/down, and you'll also need to hit pause/play to get the speed shift to activate. To be honest, this is not a priority to fix atm as it only affects the small number of people who create casts, and it's not that useful to speedup your own casts, is it?

**Comment by bobziroll** [comment]
@perborgen Agreed that it affects only a small portion, and it's not a huge annoyance. I do speed up my own casts when I'm doing a final run-through before considering a cast totally complete though, so for me there is still a use-case. Also sometimes when I'm answering questions that require remembering what was said in the cast, I'll sometimes listen through my own cast but would like to do so at a faster speed.
